src/components/dropdown-disclosure/components/activator/index.tsx

```diff
@@ -33,7 +33,11 @@ const DropdownDisclosureActivator = ({
 			<Button
 				{...buttonProps}
 				color={color}
-				icon={hideChevron ? null : <IconChevronDown16 />}
+				icon={
+					hideChevron ? null : (
+						<IconChevronDown16 className={s.chevronWrapper} />
```

Comment:
The chevron animation wasn't occurring when the disclosure opened. Lmk if there's a different way to do this! I'm not sure of the context why the disclosure is particular about the child type being a string and using this.

Reply:
I forgot about this!! It was a bug pointed out by a DS person recently and my brain forgot to make a ticket. :lolsob: This is exactly what needed to be done! 🎉